<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="home_page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="backgroundLogo_label">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>-10</y>
        <width>451</width>
        <height>471</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/files/images/NFLLogo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_viewConferences">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127); color:white</string>
      </property>
      <property name="text">
       <string>View Conferences</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Stadiums">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127); color:white</string>
      </property>
      <property name="text">
       <string>View Stadiums</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_TeamInfo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127); color:white</string>
      </property>
      <property name="text">
       <string>View Team Info</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="conferenceView_page">
     <widget class="QTableView" name="tableView_conferences">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; background: rgba(0, 0, 0, 127)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_viewNFC">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>80</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>70</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(255, 255, 255, 0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/files/images/NFCLogo.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_viewAFC">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>80</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>70</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(255, 255, 255, 0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/files/images/AFCLogo.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_conferences">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>20</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>CONFERENCES</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_divisions">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>200</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>DIVISIONS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_northDiv">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>260</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>43</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>NORTH</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_westDiv">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>360</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>43</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>WEST</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_southDiv">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>260</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>43</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>SOUTH</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_eastDiv">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>360</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>43</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>EAST</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_home">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
     <zorder>tableView_conferences</zorder>
     <zorder>pushButton_viewAFC</zorder>
     <zorder>pushButton_viewNFC</zorder>
     <zorder>label_conferences</zorder>
     <zorder>label_divisions</zorder>
     <zorder>pushButton_northDiv</zorder>
     <zorder>pushButton_westDiv</zorder>
     <zorder>pushButton_southDiv</zorder>
     <zorder>pushButton_eastDiv</zorder>
     <zorder>pushButton_home</zorder>
    </widget>
    <widget class="QWidget" name="stadium_page">
     <widget class="QTableView" name="tableView_Stadiums">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectColumns</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_home_2">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
     <widget class="QLabel" name="backgroundLogo_label_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>-10</y>
        <width>451</width>
        <height>471</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/files/images/NFLLogo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <zorder>backgroundLogo_label_2</zorder>
     <zorder>tableView_Stadiums</zorder>
     <zorder>pushButton_home_2</zorder>
    </widget>
    <widget class="QWidget" name="allTeamInfo_page">
     <widget class="QComboBox" name="comboBox_teamDropdown">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; background: rgba(0, 0, 0, 127)</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_allTeamInfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>831</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectColumns</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_home_3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_advancedQuery">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>0</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>Advanced Query</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="advancedQuery_page">
     <widget class="QPushButton" name="pushButton_home_4">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
     <widget class="QLabel" name="backgroundLogo_label_3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>-10</y>
        <width>451</width>
        <height>471</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/files/images/NFLLogo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_AdvancedQuery">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; background: rgba(0, 0, 0, 127)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127); color:white</string>
      </property>
      <property name="text">
       <string>Select Information Type</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127); color:white</string>
      </property>
      <property name="text">
       <string>Display Results From</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_AdvancedQueryCriteria">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>241</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_QuerySelection">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>410</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>Query</string>
      </property>
     </widget>
     <zorder>backgroundLogo_label_3</zorder>
     <zorder>pushButton_home_4</zorder>
     <zorder>comboBox_AdvancedQuery</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>listWidget_AdvancedQueryCriteria</zorder>
     <zorder>pushButton_QuerySelection</zorder>
    </widget>
    <widget class="QWidget" name="advancedQueryResults_page">
     <widget class="QPushButton" name="pushButton_home_5">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_advancedQueryResults">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>831</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectColumns</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_back">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>0</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 127)</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="backgroundField_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/files/images/NFLField.jpg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <zorder>backgroundField_label</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
